vowpalwabbit/core/src/reductions/search/search.cc

```diff
@@ -3366,6 +3366,9 @@ base_learner* VW::reductions::search_setup(VW::setup_base_i& stack_builder)
 
   cdbg << "num_learners = " << priv.num_learners << endl;
 
+  size_t ws = 1;
+  if (task_string == "sequencespan" || task_string == "multiclasstask") { ws = 2; }
```

Comment:
can you add the assert to search.cc just like the other PR

Reply:
also add a comment here referencing what is going on and mention the assert that fails

Reply:
https://github.com/VowpalWabbit/vowpal_wabbit/pull/4031

Reply:
updated this fix so removed this